script/crossbinary

```diff
@@ -1,6 +1,29 @@
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
@emilevauge Ah, good question. The answer is: Because this PR only affects `script/crossbinary` and it failed for me with bash 3.x. :-) (That version doesn't understand `-l` in `declare`.)

My suggestion would be to submit a follow-up PR that changes `/bin/bash` in the other scripts. If you'd prefer the shebang to be changed everywhere atomically, I can revert the part in this script. LMK.